R/generics.R

```diff
@@ -737,3 +738,30 @@ setMethod(
     obj
   }
 )
+
+
+# obj_round_type ---------------------------------------------------------------
+
+#' Rounding Type
+#'
+#' When called on a table-like object using the formatters framework, this method returns the
+#' rounding type of the object.
+#'
+#' @param obj (`ANY`)\cr a table-like object.
+#'
+#' @return The rounding type of the object (see [round_fmt()] for details).
+#' @rdname obj_round_type
+#' @export
+setGeneric("obj_round_type", function(obj) standardGeneric("obj_round_type"))
+
+#' @rdname obj_round_type
+#' @export
+setMethod(
+  "obj_round_type", "MatrixPrintForm", function(obj) obj@round_type
+)
+
+# obj_round_type setter ---------------------------------------------------------------
+#' @rdname obj_round_type
```

Comment:
Please update documentation to indicate that setter methods should only be created/used for pre-MatrixPrintForm objects, as resetting the rounding type after rounding occurs (which is during MPF creation) will not effect output when printing/exporting.